pkg/solidity-utils/contracts/helpers/InputHelpers.sol

```diff
@@ -98,4 +98,30 @@ library InputHelpers {
             previous = current;
         }
     }
+
+    /**
+     * @dev Sort an array of addresses, mutating in place (and also returning them).
+     * This assumes the tokens have been (or will be) validated elsewhere for length
+     * and non-duplication. All this does is the sorting.
+     *
+     * A bubble sort should be gas- and bytecode-efficient enough for such small arrays.
+     * Could have also done "manual" comparisons for each of the cases, but this is
+     * about the same number of operations, and more concise.
+     *
+     * This is less efficient for larger token count (i.e., above 4), but such pools should
+     * be rare. For now, sorting is only used to remove liquidity from nested pools in the
+     * batch router.
+     */
+    function sortAddresses(address[] memory addresses) internal pure returns (address[] memory) {
```

Comment:
Can we just reuse the existing `sortTokens`? (Or generalize sortTokens to sortAddresses?)

Reply:
Done!